zerver/actions/message_edit.py

```diff
@@ -206,6 +208,7 @@ def send_message_moved_breadcrumbs(
     user_mention = silent_mention_syntax_for_user(user_profile)
     old_topic_link = f"#**{old_stream.name}>{old_topic}**"
     new_topic_link = f"#**{new_stream.name}>{new_topic}**"
+    moved_message_link = stream_topic_near_url(new_stream, new_topic, target_message_id)
```

Comment:
I don't see why you didn't either do what I suggested here or comment on why you didn't -- it seems you did some other thing that is creating a fake dictionary?

Reply:
I see, it's because of the function accepting a weird type. I think it's probably OK to merge this as is, and I opened https://github.com/zulip/zulip/issues/25021 for cleaning up the system around this.